reference/promotion_settings.v3.yml

```
@@ -0,0 +1,309 @@
+openapi: "3.0.3"
+info:
+  title: Promotion Settings API
+  version: "1.0"
+  description: |
+    A *promotion setting* determines how promotions should be applied and/or calculated.
+
+    To learn more about promotions, consult the [Promotions Overview](/docs/store-operations/promotions).
+  termsOfService: "https://www.bigcommerce.com/terms"
+  contact:
+    name: BigCommerce
+    url: "https://www.bigcommerce.com"
+    email: support@bigcommerce.com
+servers:
+  - url: "https://api.bigcommerce.com/stores/{store_hash}/v3"
+    variables:
+      store_hash:
+        default: store_hash
+        description: Permanent ID of the BigCommerce store.
+    description: BigCommerce API Gateway
+security:
+  - X-Auth-Token: []
+tags:
+  - name: Promotion Settings
+paths:
+  "/promotions/settings":
+    parameters:
+      - $ref: "#/components/parameters/Accept"
+    get:
+      summary: Get Global Promotion Settings
+      description: Returns global promotion settings.
+      operationId: getGlobalPromotionSettings
+      tags:
+        - Promotion Settings
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/GlobalPromotionSettings"
+              example:
+                promotions_triggered_by_products_with_zero_product_price: true
+                promotions_apply_on_products_with_custom_product_price: false
+                number_of_coupons_allowed_at_checkout: 1
+                promotions_applied_on_original_product_price: true
+        "400":
+          description: Bad request. Authentication Required.
```

"Authentication Required" probably should be 401

```suggestion
          description: Bad request.
```